server/index.js

```diff
@@ -51,6 +51,40 @@ app.get('/api/home/:projectId', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/projects/:projectId', (req, res, next) => {
+  const projectId = Number(req.params.projectId);
+  if (!projectId) {
+    throw new ClientError(400, 'projectId must be a positive integer');
+  }
+
+  const sql = `
+    select "taskId",
+        "projectId",
+        "projectName",
+        "statusId",
+        "statusName",
+        "taskName",
+        "username",
+        "dateCreated"
+    from "tasks"
+    join "project" using ("projectId")
+    join "statuses" using ("statusId")
+    join "users" using ("userId")
+    where "projectId" = ($1)
+  `;
+
+  const param = [projectId];
+
+  db.query(sql, param)
+    .then(result => {
+      if (!result.rows) {
+        throw new ClientError(404, `cannot find projectId ${projectId}`);
+      }
```

Comment:
`result.rows` is an array and arrays are truthy. Also, an empty list is not a `404`. "All" actually means "zero or more".